packages/apps/kubernetes/templates/cluster.yaml

```diff
@@ -87,9 +87,11 @@ metadata:
 spec:
   dataStoreName: "{{ $etcd }}"
   addons:
+    {{- if not .Values.disabledDNSAddon }}
     coreDNS:
       dnsServiceIPs:
       - 10.95.0.10
+    {{- end }}
```

Comment:
Will it remove existing cordns deployment if we switch `disabledDNSAddon` from `false` to `true`?